packages/census/utils.js

```diff
@@ -0,0 +1,32 @@
+const Os = Npm.require('os');
+
+Utils = {
+  // Sends a non-blocking request
+  request(method, url, options, cb) {
+    options = _.clone(options);
+
+    HTTP.call(method, url, options, Meteor.bindEnvironment((err, result) => {
+      // If attempts are still left, keep trying
+      if (err && options.attempts--)
+        Utils.request(options, cb);
+      // If request has been succeeded or no more attempts left, invoke callback
+      else
+        cb(err, result);
+    }));
+  },
+
+  // Gets the ip address
+  ip() {
```

Comment:
Shouldn’t be a “headache for enterprise people” because you can remove all the tracking code with `meteor remove census`.